tests/json/schema-1-response/input.py

```diff
@@ -0,0 +1,326 @@
+      0.0,
+      0.0
+    ]
+  ],
+  "CC2 OPTICAL ROTATION TENSOR (VEL) @ 0NM": [
+    [
+      0.0,
+      0.0124954,
+      0.0
+    ],
+    [
+      -0.25085903,
+      0.0,
+      0.0
+    ],
+    [
+      0.0,
+      0.0,
+      0.0
+    ]
+  ],
+  "CC2 OPTICAL ROTATION TENSOR (VEL) @ 355NM": [
+    [
+      0.0,
+      -0.01673706,
+      0.0
+    ],
+    [
+      -0.26353427,
+      0.0,
+      0.0
+    ],
+    [
+      0.0,
+      0.0,
+      0.0
+    ]
+  ],
+  "CC2 OPTICAL ROTATION TENSOR (VEL) @ 439NM": [
+    [
+      0.0,
+      -0.00509424,
+      0.0
+    ],
+    [
+      -0.25891501,
+      0.0,
+      0.0
+    ],
+    [
+      0.0,
+      0.0,
+      0.0
+    ]
+  ],
+  "CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 0 @ 355NM": [
+    [
+      0.0,
+      0.0,
+      0.38619772
+    ],
+    [
+      0.0,
+      0.0,
+      0.0
+    ],
+    [
+      0.38619772,
+      0.0,
+      0.0
+    ]
+  ],
+  "CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 0 @ 439NM": [
+    [
+      0.0,
+      0.0,
+      0.37061096
+    ],
+    [
+      0.0,
+      0.0,
+      0.0
+    ],
+    [
+      0.37061096,
+      0.0,
+      0.0
+    ]
+  ],
+  "CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 1 @ 355NM": [
+    [
+      0.0,
+      0.0,
+      0.0
+    ],
+    [
+      0.0,
+      0.0,
+      10.97015701
+    ],
+    [
+      0.0,
+      10.97015701,
+      0.0
+    ]
+  ],
+  "CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 1 @ 439NM": [
+    [
+      0.0,
+      0.0,
+      0.0
+    ],
+    [
+      0.0,
+      0.0,
+      10.74076995
+    ],
+    [
+      0.0,
+      10.74076995,
+      0.0
+    ]
+  ],
+  "CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 2 @ 355NM": [
+    [
+      -5.65001748,
+      0.0,
+      0.0
+    ],
+    [
+      0.0,
+      5.62557634,
+      0.0
+    ],
+    [
+      0.0,
+      0.0,
+      0.02444115
+    ]
+  ],
+  "CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 2 @ 439NM": [
+    [
+      -5.51540341,
+      0.0,
+      0.0
+    ],
+    [
+      0.0,
+      5.49921728,
+      0.0
+    ],
+    [
+      0.0,
+      0.0,
+      0.01618613
+    ]
+  ]
+}
+
+# Convert lists to np arrays
+arrays = [
+    'CC2 DIPOLE', 'CC2 QUADRUPOLE', 'CC2 DIPOLE POLARIZABILITY TENSOR @ 355NM',
+    'CC2 DIPOLE POLARIZABILITY TENSOR @ 439NM', 'CC2 OPTICAL ROTATION TENSOR (MVG) @ 355NM',
+    'CC2 OPTICAL ROTATION TENSOR (MVG) @ 439NM', 'CC2 OPTICAL ROTATION TENSOR (VEL) @ 0NM',
+    'CC2 OPTICAL ROTATION TENSOR (VEL) @ 355NM', 'CC2 OPTICAL ROTATION TENSOR (VEL) @ 439NM',
+    'CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 0 @ 355NM',
+    'CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 0 @ 439NM',
+    'CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 1 @ 355NM',
+    'CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 1 @ 439NM',
+    'CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 2 @ 355NM',
+    'CC2 QUADRUPOLE POLARIZABILITY TENSOR COMPONENT 2 @ 439NM'
+]
+for a in arrays:
+    expected_response[a] = np.asarray(expected_response[a])
```

Comment:
```suggestion
expected_response = {k: (np.asarray(v) if isinstance(v, List) else v) for k, v in expected_response.items()}
```

Reply:
wow, that's really nifty

Reply:
`List` needs to be `list`, oops. now test is passing again 👍